firk/README.md

```diff
@@ -0,0 +1,3 @@
+# Efficient Implicit Runge-Kutta Implementation
+
+The PDF is at output/main.pdf.
```

Comment:
add the pdf

Reply:
I want to add the PDF after finishing and checking it, so that I am not going to introduce much binary to the repo.